extension/storage/filestorage/README.md

```diff
@@ -133,3 +133,29 @@ The schedule for this feature gate is:
 - Removed three releases after `stable`.
 
 [unicode_chars]: https://en.wikipedia.org/wiki/List_of_Unicode_characters
+
+## Troubleshooting
+
+When troubleshooting components that use file storage, it is sometimes helpful to read the raw contents of
```

Comment:
Can we add a note that this strategy is subject to change.

The context here is that the underlying storage mechanism, aside from being a file, is not part of an API which we have ever guaranteed. Some suggestions have been made in the past to evaluate other key/value stores which may be more performant than the bbolt one we currently use. If we ever change, we would of course have to ensure migration is possible for some time, but ultimately we should be careful not to appear locked into the implementation.

Reply:
Added in 4a32823b11.